eo-maven-plugin/pom.xml

```diff
@@ -161,14 +175,26 @@ SOFTWARE.
         <executions>
           <execution>
             <id>default-compile</id>
+            <phase>none</phase>
+          </execution>
+          <execution>
+            <id>default-testCompile</id>
+            <phase>none</phase>
+          </execution>
+          <execution>
+            <id>java-compile</id>
+            <phase>compile</phase>
+            <goals> <goal>compile</goal> </goals>
```

Comment:
@nlchar try to run `mvn clean install -Pqulice`. I'm sure it will break here, due to XML formatting mistake. Don't forget to install `xcop` first: https://github.com/yegor256/xcop

Reply:
Yegor, you have xcop and Qulice checks embedded to the CI/CD of pull requests. So these are passed. I've reformatted it anyway. :+1: